packages/web-components/src/appointment-bookings/client/components/appointment-bookings.svelte

```diff
@@ -0,0 +1,110 @@
+<script>
+  import { weekStore } from '../core/store/week-store'
+  import { themeStore } from '../core/store/theme-store'
+  import Fa from 'svelte-fa'
+  import ClickOutSide from './click-out-side.svelte'
+  import {css} from 'emotion'
+
+  import { faChevronRight, faChevronLeft } from '@fortawesome/free-solid-svg-icons'
+  import { resetCSS, generateThemeClasses } from '../../../common/styles'
+  import { generateAppointmentBookingThemeClasses } from '../core/theme'
+
+  export let theme
+  export let parentSelector
+
+  // TBC
+  // export let variant
+
+  let isModalOpen = false
+
+  let toggleModal = e => {
+    isModalOpen = !isModalOpen
+
+    // click event of component 'ClickOutSide' is added before this event buble up -> break toggle behavior
+    e.stopPropagation()
+  }
+
+  const themeClasses = {
+    ...generateThemeClasses(theme, parentSelector),
+    ...generateAppointmentBookingThemeClasses(theme, parentSelector),
+  }
+  const { button, svgNavigation } = themeClasses
+
+  themeStore.set(themeClasses)
```

Comment:
This may cause a problem as set will not update until the next tick so components won't receive the changes

Reply:
You can use store.update if this is a problem